storage/innobase/btr/btr0sea.cc

```diff
@@ -101,6 +108,32 @@ btr_search_get_n_fields(
 	return(n_fields + (n_bytes > 0 ? 1 : 0));
 }
 
```

Comment:
This does not look very clean, the function is static to this file and has empty body in release build. The compiler should be able to optimise the call away completely.